### ISSUE TITLE
User-defined car settings exposed as properties

### ISSUE THREAD

**Issue by Manu343726** [SHWotever/SimHub#771]
First of all, thank you for all your work. I find amazing how complete feature-wise SimHub is.

I'm currently working on a multiple-window dash to suit all my telemetry needs, with all tire, engine, timing, and leaderboard stats in one place. While working on the tire stats window I noticed SimHub does not expose the max rated temperature of the current tire compound, which will come in handy to correctly map temps to the desired color gauge independently from the car.

Since this property is very specific I'm 100% sure no game exposes this data (or if temp is computed from pressure, no game exposes max pressure). So I thought about manually setting this in the SimHub car settings form.

So, TL;DR, what do you think about allowing users to write custom car settings properties and expose them in the telemetry?
